src/http-client.ts

```diff
@@ -46,26 +46,14 @@ export interface IBanditParametersResponse {
   bandits: Record<string, BanditParameters>;
 }
 
-/**
- * Fixes issues with url.toString() in older React Native versions
- * that leaves a trailing slash in pathname
- * @param url URL to stringify
- * @returns stringified url
- */
-const urlWithNoTrailingSlash = (url: URL) => {
-  // Note: URL.pathname does not exist in some React Native JS runtime
-  //       so we have to do string manipulation on the stringified URL
-  return url.toString().replace(/\/\?/, '?').replace(/\/$/, '');
-};
```

Comment:
Removing the trailing-slash band-aid fix. It worked for this situation, but doesn't account for future usages of URL.